data/FilterList.json

```diff
@@ -5504,21 +5504,6 @@
     "updatedDate": "2018-12-18T16:21:12",
     "viewUrl": "https://raw.githubusercontent.com/CHEF-KOCH/CKs-FilterList/master/filters/torrent-tracker.txt"
   },
-  {
-    "id": 455,
-    "description": "HOSTS file to block all known NSA / GCHQ / C.I.A. / F.B.I. spying servers.",
-    "descriptionSourceUrl": "https://raw.githubusercontent.com/CHEF-KOCH/CKs-FilterList/master/filters/nsablocklist.txt",
-    "donateUrl": "https://github.com/CHEF-KOCH/Donations",
-    "emailAddress": "CHEF-KOCH@protonmail.com",
-    "homeUrl": "https://github.com/CHEF-KOCH/CKs-FilterList",
-    "issuesUrl": "https://github.com/CHEF-KOCH/CKs-FilterList/issues",
-    "licenseId": 17,
-    "name": "CHEF-KOCH's NSABlocklist FilterList",
-    "publishedDate": "2018-05-17T15:44:44",
-    "syntaxId": 1,
-    "updatedDate": "2018-12-30T15:04:23",
-    "viewUrl": "https://raw.githubusercontent.com/CHEF-KOCH/CKs-FilterList/master/filters/nsablocklist.txt"
```

Comment:
This list seems to still exist, but has been moved to: https://raw.githubusercontent.com/CHEF-KOCH/CKs-FilterList/master/Anti-Corp/filters/nsablocklist.txt

I suspect it is similar with the others? I'd rather wait until someone has the time to fix the broken urls in a PR than remove these all entirely.

Thanks.

Reply:
fuck. mistake

Reply:
no worries. thanks for contributing!